bors.toml

```diff
@@ -1,3 +1,3 @@
-status = ["workflow"]
-pr_status = ["workflow"]
+status = ["ci/circleci: build"]
+pr_status = ["ci/circleci: build", "workflow"]
```

Comment:
what is this doing?

Reply:
There's a `ci/circleci: build` job always expected and required but never executed, because I removed it.

![image](https://github.com/3scale/apisonator/assets/17052241/4b34e65a-94de-498a-b75c-5c388286fee2)

I thought it would disappear when I remove it from `bors.toml`, but it didn't work.

Reply:
in repo settings probably

Reply:
Yeah, it was there, thanks.